test/e2e/network/netpol/model.go

```diff
@@ -155,9 +161,10 @@ func (ns *Namespace) LabelSelector() map[string]string {
 // Pod is the abstract representation of what matters to network policy tests for
 // a pod; i.e. it ignores kube implementation details
 type Pod struct {
-	Namespace  string
-	Name       string
-	Containers []*Container
+	Namespace     string
+	Name          string
+	Containers    []*Container
+	enableWindows bool
```

Comment:
enableWindows I think this var can be global variable. Then we dont need to store it.  Or Is our test support mix kind node  cluster ? like some node is window and some are linux?

Reply:
Thats a good point there willl be no mixed nodes from now. But let's leave in the model so it's get ready when it happens.

Reply:
Yeah agreed with @knabben this makes sense to have in the model 👍